core/src/main/java/io/crate/Constants.java

```diff
@@ -47,7 +47,15 @@
 
     public static final Predicate<CharSequence> INVALID_COLUMN_NAME_PREDICATE = Predicates.contains(Pattern.compile("[\\[\\'\\]]"));
 
-    public static final String KILLED_MESSAGE = "KILLED";
-
-    public static final String NULL = "null";
+    public static class Strings {
```

Comment:
are Constants included in the client and if so, can we move these constants here to server-only?